page_analyzer/utils/url.py

```diff
@@ -18,3 +19,9 @@ def normalize(url_name: str) -> str:
         port = ":" + str(p.port)
         normalized = normalized.replace(port, "")
     return normalized
+
+
+def get_url_status_code(url_name: str) -> int:
+    r = requests.get(url_name)
+    response_code = r.status_code
+    return response_code
```

Comment:
there is no need to perform any real checks at this step, let's remove it from this pr and store it for the next step